build-integrations/sbt-1.0/project/Integrations.scala

```diff
@@ -9,8 +9,8 @@ object Integrations {
   val MiniBetterFiles = RootProject(uri(
     "git://github.com/scalacenter/mini-better-files.git#0ed848993a2fd5a36e4366b5efb9c68dce958fc2"))
   val WithResources = RootProject(
-    uri("git://github.com/scalacenter/with-resources.git#7529b2c3ac455cbb1889d4791c4e0d4957e29306"))
+    uri("git://github.com/scalacenter/with-resources.git#f0a46830cae7ef6282d9bba64b6da34bae18f339"))
```

Comment:
These new hashes contain modifications to the dummy code to fail if the env variable `BLOOP_OWNER` is not accessible via `sys.env`.